koku/api/tags/queries.py

```diff
@@ -168,11 +168,17 @@ def _build_namespace_filters_from_category_list(self, category_list):
             raise NoNamespacesForCategory(f"No namespaces for category: {category_list}")
 
         for namespace in namespaces:
-            namespace_formated = namespace.replace("%", "")  # django is doing `kube-\%%`
-            namespace_filter = QueryFilter(
-                parameter=namespace_formated, **{"field": "namespace", "operation": "startswith"}
-            )
-            category_filters.add(namespace_filter)
+            if "%" in namespace:
+                namespace_formated = namespace.replace("%", "")  # django is doing `kube-\%%`
+                namespace_filter = QueryFilter(
+                    parameter=namespace_formated, **{"field": "namespace", "operation": "startswith"}
+                )
+                category_filters.add(namespace_filter)
+            else:
+                namespace_filter = QueryFilter(
+                    parameter=namespace_formated, **{"field": "namespace", "operation": "exact"}
+                )
+                category_filters.add(namespace_filter)
```

Comment:
This could be simplified a bit.
```suggestion
            operation = "exact"
            if namespace.endswith("%"):
                namespace = namespace.replace("%", "")  # django is doing `kube-\%%`
                operation = "startswith"
                
            namespace_filter = QueryFilter(parameter=namespace, field="namespace", operation=operation)
            category_filters.add(namespace_filter)
```

Reply:
Ohhhh so clean 👍 Thanks for the suggestion Sam!